test/specs/elements/Input/Input-test.js

```diff
@@ -63,6 +63,7 @@ describe('Input', () => {
   common.implementsHTMLInputProp(Input, {
     alwaysPresent: true,
     shorthandDefaultProps: { type: 'text' },
+    strictAssert: false,
```

Comment:
How about `assertExactMatch` or similar?  It is not clear _what_ is being strictly asserted.